contracts/implementations/Mapper.sol

```diff
@@ -54,6 +54,20 @@ contract Mapper {
         ];
     }
 
+    /// @notice Gets the link between two nodes (source -> target)
+    /// @param idProxyAddressSource The address of the NFT proxy source
+    /// @param idProxyAddressTarget The address of the NFT proxy target
+    /// @param sourceNode The node Id source to be queried
+    function getNodeLink(
+        address idProxyAddressSource,
+        address idProxyAddressTarget,
+        uint256 sourceNode
+    ) external view returns (uint256 targetNode) {
+        targetNode = MapperStorage.getStorage().vehicleVirtualDeviceLinks[
```

Comment:
The general-seeming `getNodeLink` accessing the very specific `vehicleVirtualDeviceLinks` seems odd to me. Maybe call it `getStorage().nodeLinks`?

Obvious the distinction between "link" and "node link" is a bit of an implementation detail that's bubbling up because we won't migrate the old storage. I don't have a solution for _that_ besides being super verbose.

Reply:
Yeah yeah, I created that to be generic and did not generalized the variable name. Will change that.

I regret a bit about the existing links variable. Maybe we should migrate it? I don't know, does not seem to have a strong argument for this, but I would do in the future for uniformity.